financial-connections/src/main/java/com/stripe/android/financialconnections/image/StripeImageLoader.kt

```diff
@@ -0,0 +1,131 @@
+package com.stripe.android.financialconnections.image
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.util.Log
+import java.io.InputStream
+import java.net.URL
+
+/**
+ * Image loader that fetches images from memory, disk or network and runs
+ * cache policy accordingly.
+ *
+ * [StripeImageLoader] is stateful as it holds the memoryCache instance. For
+ * memory cache to work the image loader instance needs to be shared.
+ */
+internal class StripeImageLoader(
+    context: Context
+) {
+    private var currentInputStream: InputStream? = null
+
+    private val diskCache = ImageLruDiskCache(
+        context = context,
+        uniqueName = "financial_connections_image_cache"
+    )
+    private val memoryCache = ImageLruMemoryCache()
+
+    fun load(
+        url: String,
```

Comment:
probably not required now, but would be great to be able to load `Uri`, which could point to a local asset as well as a remote https image